docs/sources/reference/components/prometheus/prometheus.exporter.cloudwatch.md

```diff
@@ -257,6 +262,46 @@ require `Resource`, `Service`, `Class`, and `Type` dimensions to be specified. T
 metrics,
 all dimensions attached to a metric when saved in CloudWatch are required.
 
+### custom_namespace block
+
+The `custom_namespace` block allows the component to scrape CloudWatch metrics from custom namespaces using only the namespace name and a list of metrics under that namespace.
+For example:
+
+```alloy
+prometheus.exporter.cloudwatch "discover_instances" {
+    sts_region = "eu-west-1"
+
+    custom_namespace "customEC2Metrics" {
```

Comment:
Hmm, does this block need a label? Is it used in any way since there is `namespace` attribute in it?
```suggestion
    custom_namespace {
```

Reply:
The label name corresponds to the `Name` attribute [upstream](https://github.com/nerdswords/yet-another-cloudwatch-exporter/blob/master/docs/configuration.md#custom_namespace_job_config).